src/main/java/org/dinosaur/foodbowl/domain/store/persistence/SchoolRepository.java

```diff
@@ -1,12 +1,16 @@
 package org.dinosaur.foodbowl.domain.store.persistence;
 
+import java.util.List;
 import java.util.Optional;
 import org.dinosaur.foodbowl.domain.store.domain.School;
+import org.dinosaur.foodbowl.domain.store.domain.vo.SchoolName;
 import org.springframework.data.repository.Repository;
 
 public interface SchoolRepository extends Repository<School, Long> {
 
-    Optional<School> findByName_Name(String name);
+    Optional<School> findByName(SchoolName name);
```

Comment:
파라미터명 `schoolName`으로 맞춰주시면 감사하겠습니다 ~
```suggestion
    Optional<School> findByName(SchoolName schoolName);
```